watertap/examples/flowsheets/full_treatment_train/analysis/tests/test_multi_sweep.py

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+import pytest
+from watertap.examples.flowsheets.full_treatment_train.analysis.multi_sweep import *
+
+pytest_parameterize_list = []
+for case_num in [1, 2, 3, 4, 8, 9]:
```

Comment:
Last question: what's going on with cases 5, 6, and 7?

Reply:
Cases 4, 5, 6, and 7 seem to be the the same flow sheet: NF Two stage, and the multi_sweep tests add a good amount of time to the pytests. So in the interest of keeping the time elapsed for pytests in check, I chose to exclude them. I can include them if we don't mind the additional test time.

Reply:
It looks like we should include case 5, based on that criteria: case 4 uses `flowsheet_NF`, and case 5, 6 and 7 use `flowsheet_NF_two_stage`.

So, maybe we should include case 6, since it only has a single parameter to sweep? And leave a comment in this file explaining why we're skipping 5 and 7?